x/bank/migrations/v044/keys.go

```diff
@@ -0,0 +1,16 @@
+package v044
```

Comment:
we should remove all `x/bank/migrations` files, they are already in `x/bank/legacy`